operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -53,11 +53,31 @@ type CentralSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName=Customizations,order=6,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced"}
 	Customize *CustomizeSpec `json:"customize,omitempty"`
 
+	// Monitoring configuration.
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=1
+	Monitoring *GlobalMonitoring `json:"monitoring,omitempty"`
+
 	// Miscellaneous settings.
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName=Miscellaneous,order=7,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced"}
 	Misc *MiscSpec `json:"misc,omitempty"`
 }
 
+// GlobalMonitoring defines settings related to monitoring.
+type GlobalMonitoring struct {
+	OpenShiftMonitoring *OpenShiftMonitoring `json:"openshift,omitempty"`
+}
+
+// OpenShiftMonitoring define settings related to OpenShift Monitoring
+type OpenShiftMonitoring struct {
+	Enabled *bool `json:"enabled,omitempty"`
```

Comment:
Should we make this a `urn:alm:descriptor:com.tectonic.ui:booleanSwitch`?

Reply:
How about 
```go
//+kubebuilder:validation:Default=true
//+kubebuilder:default=true
```

Reply:
That as well, but I would suggest to start with `default=false`, and set it `default=true` later after we are sure everything works as expected.